docs/geedocs/answer/6357802.html

```diff
@@ -8,40 +8,16 @@
 <p class="sidebar-toc"><a href="../answer/6010060.html">Clearing the Google Earth EC cache</a></p>
 <p class="sidebar-toc"><a href="../answer/173203.html">EC bandwidth requirements</a></p>
 <p class="sidebar-toc"><a href="../answer/6357797.html">Select, add, or change Earth EC servers</a></p>
-<p class="sidebar-toc"><a href="../answer/2760555.html">EC 6.2 search-related changes</a></p>
 <p class="sidebar-toc"><a href="../answer/6357802.html" class="current-file">Fix Earth EC issues</a></p>
   </div>
   <div class="content">
 <a name="top_of_file"></a>
 <p><img src="../art/common/googlelogo_color_260x88dp.png" width="130" height="44" alt="Google logo" /></p>
 <h1><a href="../index.html">Google Earth Enterprise Documentation Home</a> | Google Earth Enterprise client</h1>
+
 <h2>Fix Earth EC issues</h2>
-<p>You can find solutions to most Earth EC installation and use issues in the Google Earth (free) help:</p>
-<ul>
-<li><a href="https://support.google.com/earth/topic/2376075">Installing and uninstalling</a></li>
-<li><a href="https://support.google.com/earth/topic/2376014">Crashing and graphics issues</a></li>
-<li><a href="https://support.google.com/earth/topic/2376015">Data and imagery</a></li>
-<li><a href="https://support.google.com/earth/topic/2376076">Contacting Google Earth servers</a></li>
-</ul>
-<h5>
-<h2>Issues with Earth EC installation and setup</h2>
-</h5>
-<div>
-<ul>
-<li><a href="../answer/6357797.html">Select an Earth EC server</a></li>
-<li><a href="https://support.google.com/earth/answer/2888052">Trouble downloading Earth EC to a Mac</a></li>
-<li><a href="../answer/6363126.html">Troubleshoot connection to GEE Server</a></li>
-</ul>
-</div>
-<h5>
-<h2>Issues with Earth EC features</h2>
-</h5>
-<div>
-<p>Since Earth EC include all the features of Google Earth (free version), most of the Google Earth instructions and tutorials also apply to Earth EC. For help with Earth EC features, see below.</p>
-<ul>
-<li><a href="https://support.google.com/earth/topic/2376764">Use Earth Pro and Earth EC features</a></li>
-<li><a href="https://support.google.com/earth/answer/166087">Movie Maker isn't working on Snow Leopard</a></li>
-<li><a href="https://support.google.com/earth/answer/3306088">Tips and solutions for importing files</a></li>
-</ul>
+Please see this <a href="https://productforums.google.com/forum/#!categories/maps/google-earth"> page </a> for help with Earth EC issues.
```

Comment:
Would you remove the extra spaces inside the link around the word "page"?  It's a small thing, but it looks a bit funny.